Spinner/Spinner.swift

```diff
@@ -84,25 +84,23 @@ public class SpinnerView: NSObject, Spinner {
         let view = showSpinner(inView: button, style: style, color: color)
         
         if let spinnerView = view as? SpinnerView {
-            spinnerView.userInteractionEnabledAtReception = button.isUserInteractionEnabled
```

Comment:
Sure, I haven't added a custom spinner to the Demo yet actually so I'll do that now